src/components/Movies.js

```diff
@@ -0,0 +1,51 @@
+import { useMoviesData } from "../hooks/useMoviesData"
+import { Movie } from "./Movie"
+
+export const Movies = () =>{
+    const {isLoading,data,isError,error} = useMoviesData()
+    if(isLoading){
+        return <h2>Loading...</h2>
+    }
+    if(isError){
+        return <h2>{error.message}</h2>
+    }
+    return (
```

Comment:
you need not again filter out movirs here. You are anyhow getting all movie data from useMoviesData, just show them directly. Use antd's card to show each card of movie